.github/workflows/v8upgrade.yml

```diff
@@ -0,0 +1,52 @@
+name: V8 Upgrade
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Run every day
```

Comment:
Stable releases are on average every 4 weeks. Running every day seems excessive, but I guess most of the time it will just be a version check.

Reply:
> Roughly every 4 weeks a new **major** Stable release is done.

(Source: https://v8.dev/docs/release-process#stable-releases)

Emphasis on "major" was mine, they more frequently release patch version bumps.

Currently, https://omahaproxy.appspot.com shows the most recent version bump was only about a week apart:
* current_reldate: 09/30/21
* previous_reldate: 09/24/21

If we tracked the branch head for the stable channel (currently branch-heads/9.4) then that would make the changes even more frequent.  Releasing fixes that frequently might be overkill, but automating the preparation work for getting a security fix out would be useful.